rocky/tools/templatetags/ooi_extra.py

```diff
@@ -122,3 +117,17 @@ def get_user_full_name(ooi: OOI) -> str:
         return KATUser.objects.get(id=ooi.user_id).get_full_name()
     except ObjectDoesNotExist:
         return _("Unknown user")
+
+
+@register.filter
+def url_encode_query(query_str: str) -> str:
+    query = json.loads(query_str)
+    params = {
+        "ooi_type": query["ooi_types"],
+        "clearance_level": query["scan_level"],
+        "clearance_type": query["scan_profile_type"],
+        "search": query["search_string"],
+        "order_by": query["order_by"],
+        "sorting_order": query["asc_desc"],
+    }
+    return urlencode(params, True)
```

Comment:
The name of this filter name is too broad imo as the name suggests you can pass a query string with any input. Perhaps we should make it more explicit that it's used to create query parameters from a dashboard query (e.g. `url_encode_dashboard_query`?)